cmd/validator/flags/flags.go

```diff
@@ -366,6 +366,14 @@ var (
 		Value: fmt.Sprint(params.BeaconConfig().DefaultBuilderGasLimit),
 	}
 
+	// BuilderBoostFactorFlag sets a multiplier for builder block construction as a Uint64.
+	BuilderBoostFactorFlag = &cli.Uint64Flag{
+		Name: "builder-boost-factor",
+		Usage: "A multiplier for builder block construction as a Uint64 with a default of 100. This is used to prioritize local block construction over relay/builder block construction" +
```

Comment:
On the validator client, long help messages are wrapped. Ex:

```
  --force-clear-db                     Clears any previously stored data at the data directory. (default: false)
  --grpc-max-msg-size value            Integer to define max receive message call size (in bytes).
                                         If serving a public gRPC server, set this to a more reasonable size to avoid
                                         resource exhaustion from large messages.
                                         Validators with as many as 10000 keys can be run with a max message size of less than
                                         50Mb. The default here is set to a very high value for local users. (default: 2147483647)
  --log-file value                     Specifies log file name, relative or absolute.
  --log-format value                   Specifies log formatting. Supports: text, json, fluentd, journald. (default: "text")
```

Is it possible to do the  same for this flag?

Reply:
NIT: The [actual code](https://github.com/prysmaticlabs/prysm/blob/develop/beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go#L86) inverse `local-block-value-boost` and `100`.

Reply:
oops missed updating that . will fix